src/vs/workbench/parts/search/browser/openAnythingHandler.ts

```diff
@@ -308,7 +340,12 @@ export class OpenAnythingHandler extends QuickOpenHandler {
 			entry.setHighlights(labelHighlights, descriptionHighlights);
 		});
 
-		return viewResults;
+		return {
```

Comment:
Can we make this nicer? E.g. an interface which has one entry for results and another for all performance/telemetry relevant data? Maybe just a Tupel<2>?